README.md

```diff
@@ -106,6 +100,14 @@ Snap configuration options for setting env file paths:
 | Local  | `apps.<app>.envfile=<path>` |
 
 
+| Accepted characters in snap option keys                   | Mapped environment variable names |
+|-----------------------------------------------------------|-----------------------------------|
+| Lowercase letters                                         | Uppercase letters                 |
+| Numbers (not at the beginning and with lowercase letters) | Numbers                           |
+| Hyphens (surrounded by lowercase letters)                 | Underscores                       |
+
+Any characters not listed in the first column of the above table are not allowed.
+
```

Comment:
Two issues:
- Line 93 which introduces the table hasn't been moved.
- This move results in confusion. The character table is unrelated to env file options but rather there to explain the mapping between snap options and environment variable names.

Reply:
True! Fixed!  Thanks!